xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -5563,6 +5563,11 @@ core.menu.export.rtf=Export as RTF
 core.menu.export.html=Export as HTML
 core.menu.export.xar=Export as XAR
 
+#######################################
+## until 14.4.8, 14.10.1, 15.0RC1
+#######################################
+core.delete.backlinksInfo=After deleting this page, the {0,choice,0#incoming links|1#incoming link|1<incoming links} will point to an empty page. To avoid this, you can select a new target.
```

Comment:
I think we'll have to bring this key back if we show the links to check the backlinks in the Update Links hint. So I'm starting to think that it's best if we do the changes in this PR.